prov/usnic/src/usdf_cq.c

```diff
@@ -268,10 +271,18 @@ usdf_cq_sread_common(struct fid_cq *fcq, void *buf, size_t count, const void *co
 		if (ret == -EAGAIN) {
 			if (entry > (uint8_t *)buf)
 				break;
-			if (timeout >= 0 && time_spent >= timeout)
+			if (timeout_ms >= 0 && time_spent_us >= timeout_ms * 1000)
```

Comment:
I know this is correct without the extra parenthesis, but maybe to make it more clear:
```
if ((timeout_ms >= 0) && (time_spent_us >= timeout_ms * 1000))
```